composer.json

```diff
@@ -63,16 +63,11 @@
     }
   },
   "scripts": {
-    "auto-scripts": {
-      "cache:clear": "symfony-cmd",
-      "assets:install --symlink --relative %PUBLIC_DIR%": "symfony-cmd",
-      "security-checker security:check": "script"
-    },
     "post-install-cmd": [
-      "@auto-scripts"
```

Comment:
why did you remove 

> @auto-scripts

Reply:
Because I had so many errors with this scripts. And I don't need it in production